### PR TITLE
INGK-1004 Fix test

### PR DESCRIPTION
**Pull request by MartinALL91**
### Type of change

- Simplify the test_load_configuration test
- Fix the test_load_configuration test

### Tests
- [x] Run tests.

### Code formatting and linting

- [x] Use the ruff package to format the code: `ruff format ingenialink tests`.
- [x] Use the ruff package to lint the code: `ruff check ingenialink`.

### Others

- [x] Set fix version field in the Jira issue.

### CODE REVIEW

File: tests/test_servo.py [hunk 1]
```diff
@@ -231,25 +232,23 @@ def test_load_configuration(connect_to_slave, read_config, pytestconfig):
...
+        # Check if the register exists in the drive
+        try:
             value = servo.read(reg_id, subnode=subnode)
-            if dtype == "str":
-                assert value == storage
-            elif dtype == "float":
-                assert value == pytest.approx(float(storage), 0.0001)
-            else:
-                assert value == int(storage)
+        except ILIOError:
```
Comment (polfeliu) [inline tests/test_servo.py:241]: How could a config that you just saved contain a register that doesn't exist?

Comment (MartinALL91) [reply tests/test_servo.py:241]: The eoe dictionary contains Ethernet-related registers that don't exist in the EVE-XCR-E drive

Comment (polfeliu) [reply tests/test_servo.py:241]: they don't exist via ethercat?
Or they don't exist at all? If they don't exist at all create issue on VOLCANO-490

Comment (MartinALL91) [reply tests/test_servo.py:241]: The registers don't exist on the EVE-XCR-E (they do on the CAP-XCR-E). When attempting to read them, an SDO "object doesn't exist" abort error is thrown.

Comment (polfeliu) [reply tests/test_servo.py:241]: I mean if they exist on EoE. Either way I'm creating it to review: VOLCANO-615

**Review by polfeliu — APPROVED** [review]

**Review by GullinBustin — APPROVED** [review]

**Pull request merged**